code/game/objects/items/weapons/trick.dm

```diff
@@ -0,0 +1,675 @@
+	update_icon()
+
+/obj/item/weapon/trickcards/trickdeck/MouseDrop(atom/over_object)
+	var/mob/M = usr
+	if(!ishuman(usr) || usr.incapacitated() || usr.lying)
+		return
+	if(Adjacent(usr))
+		if(over_object == M && loc != M)
+			M.put_in_hands(src)
+			usr << "<span class='notice'>You pick up the deck.</span>"
+
+		else if(istype(over_object, /obj/screen))
+			switch(over_object.name)
+				if("l_hand")
+					if(!remove_item_from_storage(M))
+						M.unEquip(src)
+					M.put_in_l_hand(src)
+				else if("r_hand")
+					if(!remove_item_from_storage(M))
+						M.unEquip(src)
+					M.put_in_r_hand(src)
+				usr << "<span class='notice'>You pick up the deck.</span>"
+	else
+		usr << "<span class='warning'>You can't reach it from here!</span>"
+		
+/obj/item/weapon/trickcards/trickcardhand
+	name = "hand of cards"
+	desc = "A number of cards not in a deck, customarily held in ones hand."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "syndicate_hand2"
+	w_class = 1
+	var/list/currenthand = list()
+	var/choice = null
+	
+/obj/item/weapon/trickcards/trickcardhand/attack_self(mob/user)
+	user.set_machine(src)
+	interact(user)
+
+/obj/item/weapon/trickcards/trickcardhand/interact(mob/user)
+	var/dat = "You have:<BR>"
+	for(var/t in currenthand)
+		dat += "<A href='?src=\ref[src];pick=[t]'>A [t].</A><BR>"
+	dat += "Which card will you remove next?"
+	var/datum/browser/popup = new(user, "cardhand", "Hand of Cards", 400, 240)
+	popup.set_title_image(user.browse_rsc_icon(src.icon, src.icon_state))
+	popup.set_content(dat)
+	popup.open()
+
+obj/item/weapon/trickcards/tricksinglecard/throw_impact(mob/living/user)
+	if(!..())
+		switch(rand(1,20))
+			if(1)
+				visible_message("<span class='notice'>The card cuts through space!</span>")
+				playsound(get_turf(src), 'sound/effects/phasein.ogg', 100, 1, -1)													
+				do_teleport(user, user, 20)						
+			if(2)
+				visible_message("<span class='notice'>The card emits smoke!</span>")
+				playsound(get_turf(src), 'sound/effects/bamf.ogg', 100, 1, -1)
+				var/datum/effect_system/smoke_spread/smoke = new	
+				smoke.set_up(10, user.loc)
+				smoke.start()
+			if(3)
+				visible_message("<span class='notice'>The card shines with a metallic sheen.</span>")
+				user.adjustBruteLoss(30)
+				playsound(get_turf(src), 'sound/weapons/smash.ogg', 100, 1, -1)							
+			if(4)
+				visible_message("<span class='notice'>The card emits heat on it's surface.</span>")
+				user.adjustFireLoss(30)
+				playsound(get_turf(src), 'sound/weapons/sear.ogg', 100, 1, -1)				
+			if(5)
+				visible_message("<span class='notice'>The card emits fire.</span>")
+				playsound(get_turf(src), 'sound/effects/fire.ogg', 100, 1, -1)
+				var/turf/T = get_turf(user)
+				for(var/turf/turf in range(1,T))
+					PoolOrNew(/obj/effect/hotspot, turf)
+			if(6)
+				visible_message("<span class='notice'>The card stops time in the surrounding area.</span>")
+				anchored = 1
```

Comment:
Honestly that is really awful code, you should just make a new timestop rather than a redefinition of it.